rocketchat-core/src/main/java/com/rocketchat/core/RocketChatClient.java

```diff
@@ -144,6 +146,15 @@ public void pinMessage(String messageId, SimpleCallback callback) {
         restImpl.pinMessage(messageId, callback);
     }
 
+    public void getRoomFiles(String roomId,
```

Comment:
lets try to keep a sane order on the methods.
Rest methods firts, and then ordered: at the top user creationg, signin, and server information (specially if it is unauthenticated), and then try to group related calls together.

Reply:
I agree with you.